docs/source/posts/2022/2022-08-30-week-11-mohamed.rst

```diff
@@ -0,0 +1,43 @@
+Week 11: Improving tutorials a little
+=====================================
+
+.. post:: August 30 2022
+   :author: Mohamed Abouagour
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+This week I didn't do much work.
+
+- Fixed some issues in the hierarchical order animation support `PR`_ that we discussed during last week's meeting (mostly naming issues).
+
+- Explained the introductory tutorial a little. But it is not good suitable beginners. So, I will spend time improving tutorials this week.
```

Comment:
add `for` or rephrase the sentence
something like `But it is not good for beginners.`